chef/cookbooks/provisioner/recipes/setup_base_images.rb

```diff
@@ -25,15 +25,18 @@
 
 tftproot = node[:provisioner][:root]
 
-# This is what we could support, but this requires validation
-#discovery_arches = ["x86_64", "ia32"]
-discovery_arches = ["x86_64"]
-
 discovery_dir = "#{tftproot}/discovery"
 pxe_subdir = "bios"
 pxecfg_subdir = "bios/pxelinux.cfg"
 uefi_subdir = "efi"
 
+# This is what we could support, but this requires validation
+#discovery_arches = ["x86_64", "ppc64le", "ia32"]
+discovery_arches = ["x86_64", "ppc64le"]
+discovery_arches.select! do |arch|
+  File.exist?("#{discovery_dir}/#{arch}/initrd0.img") && File.exist?("#{discovery_dir}/#{arch}/vmlinuz0")
```

Comment:
Metrics/LineLength: Line is too long. [105/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)